src/test/8_Vault.t.sol

```diff
@@ -1732,6 +1732,10 @@ contract vaultTests is Test {
         loc = bytes32(slot);
         vm.store(address(debt), loc, addDebt);
 
+        vm.startPrank(vaultOwner);
+        vault.setLiquidator(address(vault));
```

Comment:
@alikonuk1 Think this was the problem :D, it should be vault.setLiquidator(address(liquidator));